.github/workflows/intel.yml

```diff
@@ -24,8 +24,12 @@ jobs:
       CXX: icpx
       DEBIAN_FRONTEND: noninteractive
     steps:
+      - name: Setup cmake
+        uses: jwlawson/actions-setup-cmake@v2
+        with:
+          cmake-version: '3.30.x'
```

Comment:
Did you try anything lower?

Reply:
Nope, just bumped it to latest.

Reply:
Could you try shifting back? Once known you should set cmake_minimum_version in the CMake file if Intel is detected.

Reply:
You can do what you want in the CI (e.g. 3.30) but package maintainers need something precise.

Reply:
Should be version $\geq 3.26$ https://gitlab.kitware.com/cmake/cmake/-/commit/4331f7b4bf024c20a06b28664e4f656210de7b5d

Reply:
Great. So set CMake minimum version to 3.26 only when Intel is detected.